_data/versions.yml

```diff
@@ -1359,7 +1359,7 @@ v3.0:
 
     - Two new `calicoctl` resources: [BGP Configuration](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/bgpconfig) and [Felix Configuration](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/felixconfig).
 
-    - The `calicoctl` Policy resource has been split into [Network Policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/networkpolicy) and [Global Network Policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/globalnetworkpolicy).
+    - Those using the Kubernetes API datastore can now use `calicoctl` to create, update, and delete Calico policies. To enable this, the `calicoctl` Policy resource has been split into [Network Policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/networkpolicy) and [Global Network Policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/globalnetworkpolicy).
```

Comment:
I think this should be two separate notes - the split impacts etcd users as well, and it sort of orthogonal to supporting calicoctl policies with kdd

Reply:
Ah, OK. Good point.